Tasks/AzureFileCopy/AzureFileCopy.ps1

```diff
@@ -116,7 +118,7 @@ try
 {
     # getting azure vms properties(name, fqdn, winrmhttps port)
     $azureVMResourcesProperties = Get-AzureVMResourcesProperties -resourceGroupName $environmentName -connectionType $connectionType `
-    -resourceFilteringMethod $resourceFilteringMethod -machineNames $machineNames
+    -resourceFilteringMethod $resourceFilteringMethod -machineNames $machineNames -enableRemoteDeployment $enableRemoteDeployment
```

Comment:
Get-* method trying to enable a property does not seem semantically right. Should you have separate utility method and invoke that first?